_posts/2017-12-13-Development-Setup.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+category
+title: Setup Development
+---
+This is an example of what it takes to setup up a reswitched development environment.
+
+## Requirements
+
+- Linux or WSL is recommended.
+- LLVM and Clang Version 5 or higher.
+    - Arch Linux has recent enough versions of LLVM and Clang
```

Comment:
Rewrite this to mention the explicit commands to be used. E.G.

```
- Arch Linux: `pacman -Syu clang llvm`
- Debian/Ubuntu: check install instructions at [official llvm apt](https://apt.llvm.org/)
- Fedora: ???
```

We want users to basically just have to copy paste as much as possible.